zerver/lib/test_fixtures.py

```diff
@@ -21,6 +21,33 @@
 UUID_VAR_DIR = get_dev_uuid_var_path()
 FILENAME_SPLITTER = re.compile('[\W\-_]')
 
+def run_db_migrations(**options: Any) -> None:
+    for app_config in apps.get_app_configs():
+        if module_has_submodule(app_config.module, "management"):
+            import_module('.management', app_config.name)
+
+    app_label = options['app_label'] if options.get('app_label') else None
+    db = options.get('database', DEFAULT_DB_ALIAS)
+    migration_status_file = options.get('migration_status_file', 'migration_status_test')
+    out = StringIO()
```

Comment:
This seems unused?

Reply:
oops, I guess it came along from the bottom and I forgot to clean it up :stuck_out_tongue: